core-gl-kit/autogen.kit.d/base.yml

```diff
@@ -1299,6 +1299,74 @@ gitlab_freedesktop_dirlisting:
               meson_src_configure
             }
 
+    - switcheroo-control:
+        category: sys-power
+        exclude:
+          # glib-2.76 needed for > 2.6.0
+          - "^3.*"
+        vars:
+          desc: D-Bus service to check the availability of dual-GPU
+          python_compat: python3+
+          license: GPL-3
+          slot: '0'
+          homepage: https://gitlab.freedesktop.org/hadess/switcheroo-control/
+          gitlab_user: hadess
+          gitlab_repo: switcheroo-control
+          inherit:
+              - meson
+              - python-single-r1
+              - systemd
+          iuse: gtk-doc selinux systemd test
```

Comment:
I suggest to drop test USE and statically disable them in meson. I started to remove them from gnome too. The test process must be divided from ebuild process IMHO and keep things more clean in the ebuild.
`selinux` could be removed to probably.  The support of selinux is currently not available in Macaroni.